src/index.js

```diff
@@ -0,0 +1,136 @@
+import './style.css';
+import { saveTasks, savedTasks } from './modules/edit.js';
+
+const taskInput = document.getElementById('input');
+const addButton = document.getElementById('button');
+const todoList = document.getElementById('todo-list');
+
+function updateIndexes() {
+  const taskItems = todoList.children;
+  for (let i = 0; i < taskItems.length; i += 1) {
+    const taskItem = taskItems[i];
+    const checkbox = taskItem.querySelector('input[type="checkbox"]');
+    checkbox.dataset.index = i;
+  }
+}
+
+function removeTask(index) {
+  savedTasks.splice(index, 1);
+  saveTasks();
+  updateIndexes();
+}
+
+function removeTaskElement(li) {
+  const index = Array.prototype.indexOf.call(todoList.children, li);
+  removeTask(index);
+
+  if (li.firstChild.nextSibling.checked) {
+    li.style.display = 'none';
+  } else {
+    todoList.removeChild(li);
+  }
+}
+
+function editTask(li) {
+  const taskValue = li.firstChild.nextSibling.textContent.trim();
```

Comment:
While you did an impressive job working on the project requirements you did not follow design requirements it would be nice if you followed design requirements and always pay attention to detail. Please watch this video of how your work should be [Video](https://github.com/SAZZAD1Q2/New-TO-Do-List.git).

This is your design:
![Screenshot 2023-06-10 at 4 00 51 PM](https://github.com/SAZZAD1Q2/New-TO-Do-List/assets/79506105/5132be7a-f0bc-4e1f-82fb-bdaf6dae175d)


Please stick to this design:

![Uploading Screenshot 2023-06-10 at 4.04.43 PM.png…]()